tasks/runtime.yml

```diff
@@ -43,6 +43,16 @@
   notify:
     - Restart Munge service
 
+- name: Ensure JobComp logfile exists
+  file:
+    path: "{{ openhpc_slurm_job_comp_loc }}"
+    state: touch
+    owner: slurm
+    group: slurm
+  when: openhpc_slurm_job_comp_type == 'jobcomp/filetxt'
+  register: touch
+  changed_when: "touch.diff.before == 'absent'" # fix for molecule idempotency test - ignore change to timestamps
```

Comment:
Could you use:
```
access_time: preserve
modification_time: preserve
```
as parameters to the file module to prevent the timestamps being altered in the first place?

Reply:
Much better, thanks!